commitlint.config.ts

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  extends: ['@commitlint/config-conventional'],
```

Comment:
Both `.commitlintrc.json` and `commitlint.config.ts` are required, right?

Reply:
yeah, they are must-have, in future we can edit commit lint rules, if needed